lms/djangoapps/api_manager/users/views.py

```diff
@@ -1012,6 +1009,13 @@ def get(self, request, user_id, course_id):
         if len(queryset):
             current_grade = queryset[0].grade
             proforma_grade = queryset[0].proforma_grade
+            progress_summary = json.loads(queryset[0].progress_summary)
+            grade_summary = json.loads(queryset[0].grade_summary)
+            grading_policy = json.loads(queryset[0].grading_policy)
+        else:
```

Comment:
I removed 'else' statement for couple of reasons:

1) Data consistency, since both grade and proforma grade are set to 0, recalculating everything might result in grades in summaries to be greater than 0. Thus, I've decided that setting summaries and grading policy to empty dictionary might be more appropriate.

2) The case when the 'else' statements would be executed is the case when the user-course combination doesn't exist in the grading table. Calculating the grade for a user who's not associated with a course doesn't sound like a case we want to have.